htdocs/takepos/invoice.php

```diff
@@ -678,7 +678,10 @@ function fail($message)
 		$printer = new dolReceiptPrinter($db);
 	}
 
-	$headerorder = '<html><br><b>'.$langs->trans('Place').' '.$place.'<br><table width="65%"><thead><tr><th class="left">'.$langs->trans("Label").'</th><th class="right">'.$langs->trans("Qty").'</th></tr></thead><tbody>';
+	$sql = "SELECT label FROM ".MAIN_DB_PREFIX."takepos_floor_tables where rowid=".$place;
```

Comment:
To be sure to avoid sql injection, where sql is forged with a test on a parameter that is coming un url, it is better to protect sql forged with
...rowid = ".((int)  $place);
Can you fix this ?